awx_collection/test/awx/test_job_template.py

```diff
@@ -12,13 +12,15 @@ def test_create_job_template(run_module, admin_user, project, inventory):
     module_args = {
         'name': 'foo', 'playbook': 'helloworld.yml',
         'project': project.name, 'inventory': inventory.name,
+        'extra_vars': {'foo': 'bar'},
         'job_type': 'run',
         'state': 'present'
     }
 
     result = run_module('tower_job_template', module_args, admin_user)
 
     jt = JobTemplate.objects.get(name='foo')
+    assert jt.extra_vars
```

Comment:
Maybe we could be more explicit and say `assert jt.extra_vars == ...` ?

Reply:
Oh yeah, that's exactly right, assert it's the foo-bar dict and then this should be great.